docs/reference/docs/index_.asciidoc

```diff
@@ -308,31 +308,55 @@ containing this shard. After the primary shard completes the operation,
 if needed, the update is distributed to applicable replicas.
 
 [float]
-[[index-consistency]]
-=== Write Consistency
-
-To prevent writes from taking place on the "wrong" side of a network
-partition, by default, index operations only succeed if a quorum
-(>replicas/2+1) of active shards are available. This default can be
-overridden on a node-by-node basis using the `action.write_consistency`
-setting. To alter this behavior per-operation, the `consistency` request
-parameter can be used.
-
-Valid write consistency values are `one`, `quorum`, and `all`.
-
-Note, for the case where the number of replicas is 1 (total of 2 copies
-of the data), then the default behavior is to succeed if 1 copy (the primary)
-can perform the write.
-
-The index operation only returns after all *active* shards within the
-replication group have indexed the document (sync replication).
+[[index-wait-for-active-shards]]
+=== Wait For Active Shards
+
+To improve the resiliency of writes to the system, indexing operations 
+can be configured to wait for a certain amount of active shard copies 
+before proceeding with the operation. If the requisite number of shard
+copies are not active for every shard in the index, then the write 
+operation must wait and retry, until either the requisite shard copies
+have started or a timeout occurs. By default, write operations only 
+wait for the primary shards to be active before proceeding (i.e. a value
+of `1`). This default can be overridden in the index settings dynamically
+by setting `index.write.wait_for_active_shards`. To alter this behavior 
+per operation, the `wait_for_active_shards` request parameter can be used.
+
+Valid values are `all` or any non-negative integer up to the total number
+of configured copies per shard in the index (which is the number of replicas
++ 1).  Specifying a negative value or a number greater than the number of 
+shard copies will throw an error.
+
+For example, suppose we have a cluster of three nodes, `A, `B`, and `C` and
+we create an index `index` with the number of replicas set to 3. If we 
```

Comment:
OK I see you mean `3`- perhaps add (resulting in 4 shard copies, one more copy than there are nodes)